### DISCUSSION TITLE
(question) animated tui

### DISCUSSION THREAD

**Discussion by ghost** [charmbracelet/lipgloss#79] [Q&A] [answered]
Hi guys,

Hope you are all well !

I developed a tui for parsing my nginx/apache logs with termui but I am not a bug fan of the colors available with this package ^^

So, I tried the lipgloss example but could not figure out how to animate/update the content of pans.

Here is what I did with termui that I did like to igrate to lipgloss:
https://user-images.githubusercontent.com/1963546/134805573-fa3cd18d-1d82-4758-a461-109cc4912189.mp4

How can build a similar layout with lipgloss ? Is it possible ? How to update content of my tables ?

Thanks for any insights or inputs on these questions.

Cheers,
Luc Michalski

**Comment by jon4hz** [comment, marked as answer]
Lipgloss is a styling library. 
Take a look at https://github.com/charmbracelet/bubbletea if you search for a replacement for termui.
